### ISSUE TITLE
Ocean coverage

### ISSUE THREAD

**Issue by Yaakov719** [davidson16807/tectonics.js#53]
Could you possibly add a way to edit how much of the surface is covered by liquid water?

**Comment by jannerbram** [comment]
yes. This would be useful.

**Comment by davidson16807** [comment]
Yes, we could add a slider for specifying mass of the oceans, if that helps? Just to be clear: the model assumes the oceans fill up to occupy whatever space is below a certain sea level, it doesn't really support paintbrushing oceans into place, unless you're talking about modifying the elevation maps.

**Comment by Yaakov719** [comment]
I was thinking that the slider would be a useful way to determine how deep the oceans are, as well as create maps with very different land-water ratios to earth. A slider to modify the sea level that the model works with would be super useful, or some sort of ocean depth map. If you could add something like one of those, it'd be awesome!

**Comment by ininja-ininja** [comment]
Would adjusting this slider adjust the amount of blue in the "Topographic View" mode?
